workflow/usage.rst

```diff
@@ -277,3 +294,6 @@ The links below will only be displayed when the action is allowed:
     {% if workflow_has_marked_place(post, 'to_review') %}
         <p>This post is ready for review.</p>
     {% endif %}
+
+    {# Get all the places related to the 'post' marking #}
+    {{ workflow_marked_places(post)|join(',') }}
```

Comment:
What about something like:
```twig
{% if 'waiting_some_approval' in workflow_marked_places(post) %}
    <span class="special-label"></span>
{% endif %}
```
Many places may be needed for a "can", but some times, only one to display things.